api/trace/api.go

```diff
@@ -43,15 +43,15 @@ type Tracer interface {
 	) error
 }
 
-type EndOptions struct {
+type EndConfig struct {
```

Comment:
can you please add doc for EndConfig, EndOption and WithEndTime?

Reply:
Good spot on. I was thinking about while renaming but then had left for another PR. Adding.

Reply:
Added but let me know if they could be improved.